pom.xml

```diff
@@ -81,15 +81,14 @@
         <dependency.slf4j.version>2.0.9</dependency.slf4j.version>
         <dependency.log4j.version>2.20.0</dependency.log4j.version>
         <dependency.groovy.version>3.0.19</dependency.groovy.version>
-        <dependency.tika.version>2.4.1</dependency.tika.version>
+        <dependency.tika.version>2.9.1</dependency.tika.version>
         <dependency.truezip.version>7.7.10</dependency.truezip.version>
-        <dependency.poi.version>5.2.2</dependency.poi.version>
-        <dependency.poi-ooxml-lite.version>5.2.3</dependency.poi-ooxml-lite.version>
+        <dependency.poi.version>5.2.5</dependency.poi.version>
```

Comment:
Removed `poi-ooxml-lite.version` property as it should always match the `poi` version anyways.